covidmodel.py

```diff
@@ -1313,6 +1313,11 @@ def __init__(self, num_agents, width, height, kmob, repscaling, rate_inbound, ag
         # of those interactions
         self.variant_start_times = {}
         self.variant_start = {}
+
+        # A dictionary to count the dwell time of an agent at a location; 
+        # key is (agent, x, y) and value is count of dwell time
+        self.dwell_time_at_locations = {}
+
```

Comment:
Initialize with a for loop for all positions in the grid the dictionary using the following:

```
for i,j in positions:
    dwt = poisson_rvs(self.model.model_data.avg_dwell)
    (x, y, dwt)
```